ci/pipelines/deploy-to-test.yml

```diff
@@ -504,6 +517,11 @@ groups:
       - connector-pact-tag
       - push-connector-to-staging-ecr
       - connector-db-migration
+  - name: egress
```

Comment:
Thanks for putting this in the right place alphabetically 😸 👍

Reply:
I'm quite OCD, so this was an important detail for me!